api/src/controllers/contact.ts

```diff
@@ -45,3 +47,46 @@ export const createContact = async (
     throw errorWithCode(message, 500);
   }
 };
+
+/**
+ * POST /requests
+ * @param {Express Request} req
+ * @param {Express Response} res
+ * @returns undefined
+ */
+export const inviteToOrg = async (
+  { params, body }: { params: any, body: any }, res: Response
+): Promise<void> => {
+  logger.info('createInvitationRequest')
+
+  const { githubId: recipient, organizations } = body
+
+  try {
+    logger.info(`user  approved request created for ${recipient}`)
+    const { id } = await getUserByName(recipient)
+
+    const promises = await inviteUserToOrgs(
+      id,
+      organizations,
+      recipient
+    )
+
+    await Promise.all(promises)
+
+    // this is where we could create the invitations for recipient
```

Comment:
Is this a TODO? Our current pattern includes `// TODO: (initials, sb for me) [...comment]`